cpp/modmesh/view/RAction.hpp

```diff
@@ -47,43 +47,6 @@ class RAction
         QObject * parent = nullptr);
 }; /* end class RAction */
 
-class RPythonAction
-    : public QAction
-{
-public:
-    RPythonAction(
-        QString const & text,
-        QString const & tipText,
-        QString const & appName,
-        QObject * parent = nullptr);
-
-    void run();
-
-private:
-    QString m_pyFuncName;
-}; /* end class RPythonAction */
-
-class RAppAction
```

Comment:
With action setup code ported to Python, this helper class `RAppAction` is not longer needed.